clippy_lints/src/operators/float_cmp.rs

```diff
@@ -64,7 +69,7 @@ pub(crate) fn check<'tcx>(
 
         if let Some(name) = get_item_name(cx, expr) {
             let name = name.as_str();
-            if name == "eq" || name == "ne" || name == "is_nan" || name.starts_with("eq_") || name.ends_with("_eq") {
+            if name == "eq" || name == "ne" || name.starts_with("eq_") || name.ends_with("_eq") {
```

Comment:
The `name.starts_with("eq_") || name.ends_with("_eq")` part is worrying me. Is it supposed to be for `PartialEq` methods? If so, please check the full name and that it is from `PartialEq` (same for `eq` and `ne`).

Reply:
That isn't even added in this PR.

Reply:
Indeed but the worry stays. Is there already a test for it by any chance? If so just resolve this conversation please.